ts/api/api.ts

```diff
@@ -361,7 +361,73 @@ class InitializeOnDemandApi implements OnDemandOpenYoloApi {
   }
 }
 
-export const openyolo: OnDemandOpenYoloApi = new InitializeOnDemandApi();
+/**
+ * Promise stub that immediately rejects.
+ */
+class FakePromise {
+  then(onFulfilled?: Function, onRejected?: Function): FakePromise {
+    window.setTimeout(() => {
+      onRejected(OpenYoloError.unsupportedBrowser());
```

Comment:
Oh I missed something here: if the onRejected function is not given, this will raise an error.